ledger/lruaccts.go

```diff
@@ -53,6 +55,7 @@ func (m *lruAccounts) init(log logging.Logger, pendingWrites int, pendingWritesW
 	m.pendingNotFound = make(chan basics.Address, pendingWrites)
```

Comment:
I think making channels and maps being nil are reasonable points, for only thing that worries me is to write to a channel that turn out to be nil, and it seems write to nil channel blocks forever, and it will never be selected.

Reply:
So I am changing the setting for now to take your suggestions.

Reply:
also confirmed that https://github.com/algorand/go-algorand/pull/5058/commits/f5f2af3db9a9c3742d2960274265a30766e50db8 can be triggered

Reply:
cr [922413a](https://github.com/algorand/go-algorand/pull/5058/commits/922413a1ea829352d338daab5896eff92dfd2956)